operator/test/test.sh

```diff
@@ -97,56 +97,52 @@ wait_for_pipeline() {
 }
 
 check_pod_security() {
-  sccs="$(kubectl get pod -o name -n "$1" | xargs -l -r kubectl get -o jsonpath='{.metadata.annotations.openshift\.io/scc}' -n "$1")"
-  if [[ "$sccs" =~ "restricted-v2" ]]; then
-     prune="$(echo "$sccs" | sed 's/restricted-v2//g')"
-     # if anything besides restricted-v2 is in there, we want to investigate
-     if [ -z "$prune" ]; then
-       echo "   - OK pod security for $1"
-     else
-       echo "Failed, scc's are "
-       echo "$sccs"
-       echo "[ERROR] Unexpected $1 pod security context constraints" >&2
-       exit 1
-     fi
-  else
-     # if none of the pods are restricted-v2, we want to investigate
-     echo "Failed, scc's are "
-     echo "$sccs"
-     echo "[ERROR] Unexpected $1 pod security context constraints" >&2
-     exit 1
+  mapfile -t pods < <(kubectl get pod -o name -n "$1")
+  if [[ "${#pods[@]}" -gt 0 ]]; then
```

Comment:
Is the if necessary? I would expect the for loop to not run if the array is empty...

Reply:
well ... of course there are pods in my test setup, so it works fine :-) 

let's assume you are just right here ... will push shortly